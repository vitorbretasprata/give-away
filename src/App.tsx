import React from 'react';

function App() {
  return (
    <div className="App">
      <h2>Give away</h2>
      <h2>Give away</h2>
      <h2>Give away</h2>
    </div>
  );
}

export default App;
