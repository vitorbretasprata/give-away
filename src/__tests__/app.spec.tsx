import * as React from 'react';
import { render, RenderResult } from '@testing-library/react';
import App from '../App';

let documentBody: RenderResult;

describe('<App />', () => {

  beforeEach(() => {
    documentBody = render(<App />);
  });

  it('Matches snapshot', () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});