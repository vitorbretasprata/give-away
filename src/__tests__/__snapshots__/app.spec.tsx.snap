// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Matches snapshot 1`] = `
<body>
  <div>
    <div
      class="App"
    >
      <h2>
        Give away
      </h2>
      <h2>
        Give away
      </h2>
      <h2>
        Give away
      </h2>
    </div>
  </div>
</body>
`;
